sass/px-context-browser-columns.scss

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2014 GE Global Research. All rights reserved.
+ *
+ * The copyright to the computer software herein is the property of
+ * GE Global Research. The software may be used and/or copied only
+ * with the written permission of GE Global Research or in accordance
+ * with the terms and conditions stipulated in the agreement/contract
+ * under which the software has been supplied.
+ */
+
+// Settings
+@import "px-defaults-design/_settings.defaults.scss";
+@import "px-colors-design/_settings.colors.scss";
+
+// Generic
+@import "px-normalize-design/_generic.normalize-mini.scss";
+@import "px-box-sizing-design/_generic.box-sizing.scss";
+
+// Component
+#columns > ::slotted(:not(.active):not(.animating)) {
```

Comment:
Make sure you test this after changing. Please also check the corresponding html file (px-context-browser-columns.html) and see if we use the `#columns` id in our JavaScript (`this.$.columns` or `Polymer.dom(this.root).querySelector('#columns')`). If we don't use it, please remove the id completely the HTML.